Makefile

```diff
@@ -179,7 +179,7 @@ e2e-tests: e2e-substitutions cluster-templates ## This target should be called f
 		$(CONTAINER_RUNTIME) pull $$image; \
 	done
 	time go test -v -timeout 24h -tags=e2e ./test/e2e/... -args \
-		-ginkgo.v -ginkgo.trace -ginkgo.progress -ginkgo.failFast -ginkgo.noColor=$(GINKGO_NOCOLOR) \
+		-ginkgo.timeout=6h -ginkgo.v -ginkgo.trace -ginkgo.progress -ginkgo.failFast -ginkgo.noColor=$(GINKGO_NOCOLOR) \
```

Comment:
we might not need `time` and `go test -timeout`

Reply:
Yes. I dont want to touch that in this PR. I will do a followup.